tools/macos/README.md

```diff
@@ -27,7 +27,7 @@ If you prefer to use Docker-machine, you can follow instructions in [docker-mach
 The following are required to build and deploy OpenWhisk from a Mac host:
 
 - [Docker 18.06.3](https://docs.docker.com/docker-for-mac/install/)
-- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or [Open JDK 8](https://adoptopenjdk.net/releases.html#x64_mac)
+- [Open JDK 8](https://adoptopenjdk.net/releases.html#x64_mac)
```

Comment:
We don’t need 8 anyway. I’ve been locally using 10 for a long time.

Reply:
Updated to JDK10.
BTW, I observed some certificates issues with JDK 10 in some machines.
I didn't look into it deeply yet, do you have any idea on it?